classes/class-p4-campaign-exporter.php

```diff
@@ -82,8 +81,7 @@ public function single_post_export_bulk_action() {
 		 * @param object $post Post object.
```

Comment:
Again not part of this PR, but please fix doc comments. $actions is `array`, $post is `WP_Post` and add @return array
and `public function single_post_export( $actions, $post ) : array {` in line 83